### PR TITLE
Add new flags to go stress client

### PR DESCRIPTION
**Pull request by ncteisen**
Stress client can now handle connections with TLS. It can also use the
custom ca file in the new testdata file.

### CODE REVIEW

File: stress/client/main.go [hunk 1]
```diff
@@ -60,6 +61,12 @@ var (
 	numChannelsPerServer = flag.Int("num_channels_per_server", 1, "Number of channels (i.e connections) to each server")
 	numStubsPerChannel   = flag.Int("num_stubs_per_channel", 1, "Number of client stubs per each connection to server")
 	metricsPort          = flag.Int("metrics_port", 8081, "The port at which the stress client exposes QPS metrics")
+	useTLS               = flag.Bool("use_tls", false, "Connection uses TLS if true, else plain TCP")
+	testCA               = flag.Bool("use_test_ca", false, "Whether to replace platform root CAs with test CA as the CA root")
+	tlsServerName        = flag.String("server_host_override", "foo.test.google.fr", "The server name use to verify the hostname returned by TLS handshake if it is not empty. Otherwise, --server_host is used.")
```
Comment (menghanl) [inline stress/client/main.go]: And print them in `logParameterInfo()`.

**Comment by ncteisen** [comment]
Addressed all comments

**Comment by adelez** [comment]
Can you make the flags consistent with C++ stress client? Particularly, can you define a server_port flag?

**Comment by ncteisen** [comment]
They will be consistent with cpp once my second PR goes through:
https://github.com/grpc/grpc/pull/8819

We decided that having a server port and server host flag was hacky because
is masks the server_addresses flag originally present in the stress client

**Pull request merged**